### DISCUSSION TITLE
Document suggestions

### DISCUSSION THREAD

**Discussion by dpbraune** [jdholtz/auto-southwest-check-in#165] [Ideas]
References to Python (particularly sample commands) are to python3. Suggest dropping the 3.

The CONFIG.JSON has numerous parameters but the examples never show a comprehensive example of samples for all possible parameters.  Would be a welcome addition.

Add the Examples section as part of the Table of Contents

A description of using mailto would be helpful

**Comment by jdholtz** [comment]
> References to Python (particularly sample commands) are to python3. Suggest dropping the 3.

The reason I would argue against it is that people may have the `python` command symlinked to Python 2 still (even though that would be quite rare). Even though most users probably don't have Python 2, I currently don't see a reason in doing so as it would only cause problems but not really solve anything (to my knowledge).

> The CONFIG.JSON has numerous parameters but the examples never show a comprehensive example of samples for all possible parameters. Would be a welcome addition.

All the explanations and examples can be found in the [Configuration](https://github.com/jdholtz/auto-southwest-check-in/blob/master/CONFIGURATION.md) documentation. Please let me know if anything seems unclear.

> Add the Examples section as part of the Table of Contents

What examples part is this referring to? If it is for the configuration, it already is part of the Table of Contents (the section for configuration can be found [here](https://github.com/jdholtz/auto-southwest-check-in/tree/master#configuration)).

> A description of using mailto would be helpful

This is actually implemented by [Apprise](https://github.com/caronc/apprise), not this script. In the [Notification URLs description](https://github.com/jdholtz/auto-southwest-check-in/blob/master/CONFIGURATION.md#notification-urls), it states to look at the Apprise Readme to get more information on how to set up the URLs correctly.

**Reply by jdholtz** [reply]
@dpbraune do you have any comments/suggestions to my response above?

**Comment by dmytrokoren** [comment]
@jdholtz plesse update Configuration.md file

Fare Check

Default: false **(true by default)**
Type: Boolean

**Reply by jdholtz** [reply]
Fixed in the develop branch. Thanks for pointing it out!
